src/Core/MailTemplates/Handlebars/OrganizationSeatsAutoscaled.html.hbs

```diff
@@ -1,8 +1,8 @@
 ﻿{{#>FullHtmlLayout}}
 <table width="100%" cellpadding="0" cellspacing="0"
-    style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+    style="margin: 0; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
```

Comment:
Sorry for not being explicit, but font-size, color, line-height, webkit-font-smoothing, -webkit-text-size-adjust should also be possible to remove since there's no text it can affect.